modules/webxr/webxr_interface_js.cpp

```diff
@@ -301,6 +301,16 @@ void WebXRInterfaceJS::uninitialize() {
 	};
 };
 
+Dictionary WebXRInterfaceJS::get_system_info() {
+	Dictionary dict;
+
+	// TODO get actual information from WebXR to return here
+	dict[SNAME("XRRuntimeName")] = String("WebXR");
+	dict[SNAME("XRRuntimeVersion")] = String("1.0");
```

Comment:
I'd be reluctant to say it's version "1.0", since there isn't any version currently, and maybe one day there will actually be a version "1.0" that means something specific.

So, if we're putting in filler data here (as opposed to my idea about using the browser name and version), it'd probably be best to use `String("")` like you have in the `MobileVRInterface`.

Reply:
Yeah I'll leave that out :)